src/main/java/gov/usgs/locator/LocMain.java

```diff
@@ -81,14 +90,17 @@
   public static void main(String[] args) {
     if (args == null || args.length == 0) {
       System.out.println(
-          "Usage:\nneic-locator --modelPath=[model path] --fileType=[json or hydra] "
+          "Usage:\nneic-locator --modelPath=[model path] --inputType=[json, detection, or hydra] "
               + "\n\t--logPath=[log file path] --logLevel=[logging level] "
-              + "\n\t--filePath=[input file path]"
+              + "\n\t--filePath=[input file path] [--outputType=[optional json or hydra]]"
+              + "\n\t[--locationConfig='optional config file path']"
               + "\nneic-locator --mode=batch --modelPath=[model path] "
-              + "\n\t--fileType=[json or hydra] --logPath=[log file path] "
+              + "\n\t--inputType=[json, detection, or hydra] --logPath=[log file path] "
               + "\n\t--logLevel=[logging level] --inputDir=[input directory path] "
               + "\n\t--outputDir=[output directory path] "
-              + "--archiveDir=[optional archive path] "
+              + "[--archiveDir=[optional archive path]] "
+              + "\n\t[--outputType=[optional json or hydra]] "
```

Comment:
totally minor but you have input type and output type defined multiple times in this usage, which to me implies that you could have different input and output types per directory, but since there is only one instance of these properties that seems to not be the case...so maybe check this usage comment and make sure it says what the code does...

Reply:
It's mentioned multiple times because there are several different modes the locator can be run in:

```
Usage:
neic-locator --modelPath=[model path] --inputType=[json, detection, or hydra] 
        --logPath=[log file path] --logLevel=[logging level] 
        --filePath=[input file path] [--outputType=[optional json or hydra]]
        [--locationConfig='optional config file path']
neic-locator --mode=batch --modelPath=[model path] 
        --inputType=[json, detection, or hydra] --logPath=[log file path] 
        --logLevel=[logging level] --inputDir=[input directory path] 
        --outputDir=[output directory path] [--archiveDir=[optional archive path]] 
        [--outputType=[optional json or hydra]] [--locationConfig='optional config file path']
        --csvFile=[optional csv file path]
neic-locator --mode=service
```